pom.xml

```diff
@@ -162,6 +166,7 @@
                             <goal>integration-test</goal>
                             <goal>verify</goal>
                         </goals>
+                        <phase>install</phase>
```

Comment:
Is this necessary?

Reply:
Oops. No that wasn't meant to be committed.